frontend/components/confirmation-dialog.tsx

```diff
@@ -21,8 +21,8 @@ export default function ConfirmationDialog( {description, onConfirm, isOpen, clo
     <Dialog open={isOpen}>
       <DialogContent className="[&>button:last-child]:hidden">
         <DialogHeader>
-          <DialogTitle>Bist du sicher?</DialogTitle>
-          <DialogDescription>{description}</DialogDescription>
+          <DialogTitle className={'text-3xl mb-3'}>Bist du sicher?</DialogTitle>
+          <DialogDescription className={'text-lg'}>{description}</DialogDescription>
```

Comment:
```suggestion
          <DialogTitle >Bist du sicher?</DialogTitle>
          <DialogDescription >{description}</DialogDescription>
```
I learned how to use suggestions, god damn I feel so great

"I can do that myself  H E L L O" - @calvalre